lib/helpers/redis-stubs.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var sinon = require('sinon');
+var redis = require('redis');
+var clientStub = {
+  'get': sinon.stub().callsArg(1),
+  'expire': sinon.stub().callsArg(2),
+  'del': sinon.stub().callsArg(1),
+  'set': sinon.stub().callsArg(2)
+};
+
+var redisCreateClientStub = sinon.stub(redis, 'createClient', function(){
+  return clientStub;
+});
+
+exports.redisCreateClientStub = redisCreateClientStub;
+exports.clientStub = clientStub;
```

Comment:
I tried to put this inside the test directory, but couldn't find a way to exclude it, as it's not a test.  Is there a way to exclude files from the basic ```npm test``` when that fires the ```mocha``` runner.  There are some options inside mocha itself, but they only deal with the test name.